code/modules/surgery/bodyparts/species_parts/misc_bodyparts.dm

```diff
@@ -469,6 +469,10 @@
 	is_dimorphic = FALSE
 	should_draw_greyscale = FALSE
 
+/obj/item/bodypart/head/golem/durathread/limb_unarmedattack_effect(mob/living/carbon/human/user, mob/living/carbon/human/target)
+	. = ..()
```

Comment:
you dont need to call parent here, parent is null
also when do golems attack with their head?

Reply:
If they have monkeybrain, they bite, which uses their head as the attacking limb.